### DISCUSSION TITLE
Support for vultr cloud

### DISCUSSION THREAD

**Discussion by AliSajid** [infracost/infracost#1275] [Ideas & Feature Requests]
Hi,

This is borne out of personal annoyance but what do you guys think about supporting other smaller cloud providers? I really want to use [Vultr](vultr.com).

I'd be willing to create the entire thing as they have a verified terraform provider and provide an API for pricing as well.

**Comment by alikhajeh1** [comment]
Hi @AliSajid! Thanks for creating this, please see https://github.com/infracost/cloud-pricing-api/issues/12, we'd be happy to help if you'd like to push this forward :)

**Comment by AliSajid** [comment]
Heh. I did not think to search in the cloud-pricing-api repo. It'd be lovely to help.

I'd need a little bit of guidance on this but otherwise there's only a handful of terraform resources and the flavors.

I'm a newcomer for Go so can I get an idea of which parts of [infracost/infracost](https://github.com/infracost/infracost) I'd need to modify as opposed to which parts of the [infracost/cloud-pricing-api](https://github.com/infracost/cloud-pricing-api)?
